types/dns2/index.d.ts

```diff
@@ -85,6 +85,10 @@ declare namespace DNS {
         data?: string;
     }
 
+    interface UdpDnsServerOptions {
+        type: string;
```

Comment:
I think we could tighten this up slightly:

```suggestion
        type: 'udp4' | 'udp6';
```

Looks like the type gets passed up to Node.js's APIs, which do explicitly limit to just those values.